.circleci/config.yml

```diff
@@ -518,14 +565,102 @@ workflows:
           test_apk: *salesforceReact-apk-path
           api_level: 24
 
-  test-api-27:
+  test-api-31:
+    triggers:
+      - schedule:
+          cron: "0 3 * * 6"
+          filters:
+            branches:
+              only:
+                - dev
+    jobs:
+      - run-tests:
+          name: "SalesforceAnalytics API 31"
```

Comment:
I think this will fail because Firebase Test Lab does not support API 31 yet

Reply:
Ok, I'll comment out 31 for now.